lib/fetching/content-services/dummy-subscribe-content-service.js

```diff
@@ -0,0 +1,93 @@
+// Used for testing purposes.
```

Comment:
Nope. I made this file only when I intended to write a test which would selectively stop listening to one source while listening to others. This file need not be there at all.